flask_testing/utils.py

```diff
@@ -141,14 +157,48 @@ def _post_teardown(self):
         if hasattr(self, 'templates'):
             del self.templates
 
+        if hasattr(self, 'flashed_messages'):
+            del self.flashed_messages
+
         if _is_signals:
             template_rendered.disconnect(self._add_template)
+
+        if _is_message_flashed:
+            message_flashed.disconnect(self._add_flash_message)
+        
         if hasattr(self, '_original_template_render'):
             templating._render = self._original_template_render
 
         if self.run_gc_after_test:
             gc.collect()
 
+    def assertMessageFlashed(self, message, category='message'):
+        """
+        Checks if a given message were flashed.
+        Only works if your version of Flask has signals
+        support (0.6+) and blinker is installed.
+        Only works if your version of Flask has message_flashed
+        signal support (0.10+) and blinker is installed.
+        
+        :param message: expected message
+        :param category: expected message category
+        """
+        
+        if not _is_signals:
```

Comment:
I think it would be enough to combine these into one check. `if not (_is_signals and _is_message_flashed)` and return a more cohesive error message. Similarly the docstring to this method could be more concise to explain that you need Flask 0.10+ and blinker installed in order to use this method